<launch>
  <include file="$(find kinect2_bridge)/launch/kinect2_bridge.launch"/>

  <!-- qr tracking node -->
  <node pkg="visp_auto_tracker" type="visp_auto_tracker" name="visp_auto_tracker" output="screen">
    <param name="model_path" value="$(find visp_auto_tracker)/models" />
    <param name="model_name" value="pattern" />
    <param name="debug_display" value="True" />
	      
    <remap from="/visp_auto_tracker/camera_info" to="/kinect2/hd/camera_info"/>
    <remap from="/visp_auto_tracker/image_raw" to="/kinect2/hd/image_color"/>
  </node>


  
</launch>
